src/plugins/intel_gpu/src/graph/select.cpp

```diff
@@ -31,6 +33,28 @@ layout select_inst::calc_output_layout(select_node const& node, kernel_impl_para
     return layout(in_layout.data_type, in_layout.format, output_size);
 }
 
+template<typename ShapeType>
+std::vector<layout> select_inst::calc_output_layouts(const select_node& /*node*/, const kernel_impl_params& impl_param) {
+    auto input0_layout = impl_param.get_input_layout(0);
+    auto input1_layout = impl_param.get_input_layout(1);
+    auto input2_layout = impl_param.get_input_layout(2);
+
+    auto desc = impl_param.typed_desc<select>();
+    auto dt = desc->output_data_type.value_or(input1_layout.data_type);
+
+    ov::op::v1::Select op;
+    std::vector<ShapeType> output_shapes = { ShapeType{} };
+    std::vector<ShapeType> input_shapes = {
+        input0_layout.get_partial_shape(),
```

Comment:
Why don't you use input0_layout.get<ShapeType>() instead of get_partial_shape() ?

Reply:
done